content/3. Procedurele informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/1. If-Else.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 1. If-Else
+taxonomie:
+  - IB-19.2.Controlestructuur-Beslissingen-If-Else.PI
+---
+> Meer informatie over [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/1. If-Else|If-else]]
+### Stappen in de Systematische Probleem Aanpak
+1. **Doel**: Bepaal de reden voor het gebruik van een *If-Else*-structuur. Bijvoorbeeld: Het programma moet beslissingen kunnen nemen op basis van invoer of andere data.
+
+2. **Subdoelen**: Splits het probleem in subdoelen, zoals:
+   - Het vaststellen van de conditie(s) die getoetst moet worden.
+   - Het formuleren van de actie(s) die gedaan moeten worden op basis van de conditie(s).
```

Comment:
Musing/Gedachtespinsel: Mogelijk is dit al wat vergezocht, maar een subdoel kan ook iets zijn in de trend van 'hoe gaan de condities worden getoetst'. Al kom je dan wat meer in de buurt van testen..